roxie/ccd/ccdqueue.cpp

```diff
@@ -1965,7 +2213,21 @@ class RoxieSocketQueueManager : public RoxieReceiverBase
             doFileCallback(packet);
         }
         else if ((header.activityId & ~ROXIE_PRIORITY_MASK) == 0)
-            doIbyti(header, queue, topology); // MORE - check how fast this is!
+            doIbyti(header, queue
+#ifndef SUBCHANNELS_IN_HEADER
```

Comment:
not sure I really like the prototype depending on the conditional compile.

Reply:
Nor do I, and a later refactoring of this code fixes it (probably in my next PR though)